handbook/people-ops/vacation-and-fto.md

```diff
@@ -0,0 +1,19 @@
+# Vacation: flexible time off policy
+
+Flexible Time Off (FTO) allows eligible employees to take personal time away from work for rest and recreation, vacation, medical and other personal appointments, short term illness (less than five days), or other situations.
```

Comment:
This document is very clinical, unlike our [blog post](https://about.sourcegraph.com/blog/why-vacation-at-tech-companies-should-be-mandatory-better-code-happier-people) which discusses motivation and mandatory 2 week minimum. I think this document should explain our philosophy and motivations for this policy.